com.stansassets.scene-management/Editor/Settings/SceneViewInfo.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using UnityEngine;
+
+namespace StansAssets.SceneManagement
+{
+    [Serializable]
+    class SceneViewInfo
+    {
+        public Vector3 Position;
+        public Quaternion Rotation;
+        public Vector3 Pivot;
+        public float Size;
+        public bool is2D;
+        public bool isOrtho;
```

Comment:
I don't think we need set access to those properties. Can you make it private + public get property?

Reply:
Okay